configuration/certificates.md

```diff
@@ -92,11 +92,11 @@ systemctl restart gravwell_searchagent.service
 
 ### Making browsers trust the self-signed certificate
 
-Browsers will throw up a warning if a certificate is not signed by a recognized root CA. However, we can make the browser trust our certificate by installing it manually.
+Browsers will issue a warning if a certificate is not signed by a recognized root CA. However, you can make the browser trust our certificate by installing it manually.
```

Comment:
"we" is used throughout the document, so I think we should use it here.

Reply:
"you" is used 8 times in that document, "we" is used 3.